4-Firebase-Firestore/end/firestore.indexes.json

```diff
@@ -0,0 +1,107 @@
+{
+  "indexes": [
+    {
+      "collectionGroup": "recipes",
+      "fields": [
+        {
+          "fieldPath": "category",
+          "mode": "ASCENDING"
+        },
+        {
+          "fieldPath": "unixTimeStampInMilliseconds",
```

Comment:
I'd prefer if we used `UnixTimestamp` as the field name. The milliseconds part is inferred.